server/src/server/session/TransactionOLTP.java

```diff
@@ -553,11 +512,11 @@ private TransactionException labelTaken(SchemaConcept schemaConcept) {
      * @return The {@link SchemaConcept} which was either cached or built via a DB read or write
      */
     private SchemaConcept buildSchemaConcept(Label label, Supplier<SchemaConcept> dbBuilder) {
-        if (cache().isTypeCached(label)) {
-            return cache().getCachedSchemaConcept(label);
-        } else {
+//        if (cache().isTypeCached(label)) {
```

Comment:
Decide whether to keep commented as reminder of how to re-enable Concept Type cache if we need it, or whether to delete this code.